scripts/alpine_install.sh

```diff
@@ -20,6 +20,13 @@ apk update && apk add --no-cache \
   tar \
   wget \
   zstd-dev
+  python3 \
```

Comment:
doesn't Alpine have python3 installed?

Reply:
@tigercosmos you can also just install "py3-psutil"

Reply:
@clementperon  good to know!

Reply:
it's unclear which library we will use, so I will keep these packages for now.